x/fees/keeper/querier_test.go

```diff
@@ -0,0 +1,38 @@
+package keeper_test
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/desmos-labs/desmos/x/fees/keeper"
+	"github.com/desmos-labs/desmos/x/fees/types"
+	abci "github.com/tendermint/tendermint/abci/types"
+)
+
+func (suite *KeeperTestSuite) Test_queryParams() {
+	tests := []struct {
+		name      string
+		path      []string
+		expResult types.Params
+	}{
+		{
+			name:      "Returning posts parameters correctly",
+			path:      []string{types.QueryParams},
+			expResult: types.DefaultParams(),
+		},
+	}
+
+	for _, test := range tests {
+		test := test
+		suite.Run(test.name, func() {
+			suite.SetupTest()
+
+			suite.keeper.SetParams(suite.ctx, types.DefaultParams())
```

Comment:
Instead of always using `types.DefaultParams()`, I would suggest specifying the value to be stored instead each use case, to make sure that even non-default values are returned properly